mqlc/mqlc_test.go

```diff
@@ -1783,7 +1783,7 @@ func TestSuggestions(t *testing.T) {
 		{
 			// resource suggestions
 			"ssh",
-			[]string{"os.unix.sshd", "sshd", "sshd.config"},
+			[]string{"os.unix.sshd", "sshd", "sshd.config", "windows.security.health"},
```

Comment:
I want to merge this PR so I just added the thing that is returned as a suggestion. When I run cnquery and type `ssh` in the shell I don't see any windows stuff. Not sure where the difference is with the test

Reply:
that looks correct to me, you might not be on builtin or have the old provider installed?